DSharpPlus/Enums/Permission.cs

```diff
@@ -268,6 +268,12 @@ public enum Permissions : long
         [PermissionString("Manage emoji")]
         ManageEmojis = 0x0000000040000000,
```

Comment:
```suggestion
        ManageEmojisAndStickers = 0x0000000040000000,
```

Reply:
This is more like in the docs & discord client.
![image](https://user-images.githubusercontent.com/14029133/124099352-b77a6e80-da5d-11eb-99c0-f271061a2d71.png)
https://github.com/discord/discord-api-docs/pull/3128/files#diff-83c324034ba0b1753dde3487f095f5307f7036c3e53a25bacba1879733a6ce23R61

Reply:
It's also a breaking change.

Reply:
I get u